__tests__/get-endpoints.test.js

```diff
@@ -0,0 +1,50 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data/index")
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    return db.end();
+});
+
+describe("GET /api/topics", () => {
+    test("responds with an array of topic objects with slug and description properties", () => {
+        return request(app)
+            .get("/api/topics")
+            .then((result) => {
+                expect(result.status).toBe(200);
+                expect(Array.isArray(result.body)).toBe(true);
+                result.body.forEach((topic) => {
+                    expect(topic).toHaveProperty("slug");
+                    expect(typeof topic.slug).toBe("string");
+                    expect(topic).toHaveProperty("description");
+                    expect(typeof topic.description).toBe("string");
+                });
+            });
+    });
+});
+
+test("responds with an array of topic objects with at least one topic", () => {
+    return request(app)
+        .get("/api/topics")
+        .then((result) => {
+            expect(result.status).toBe(200);
+            expect(Array.isArray(result.body)).toBe(true);
+            expect(result.body.length).toBeGreaterThan(0);
+        });
```

Comment:
Two changes I'd make with this test 

You don't need two separate tests, you can check the length in the first test. Delete this test and put a length check on line 20/21. 

Instead of checking that there's more than one, you can check for a specific number of topics. How many do you expect? Have a look at the test data 👍